crates/pgt_treesitter_grammar/grammar.js

```diff
@@ -0,0 +1,3405 @@
+/**
```

Comment:
also copied from the derek stride repo, I just added JSDoc types and fixed some linting errors